core/src/test/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoEntityProviderSpecSupport.scala

```diff
@@ -0,0 +1,123 @@
+package org.broadinstitute.dsde.rawls.entities.datarepo
+
+import java.util.UUID
+
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import bio.terra.datarepo.model.{ColumnModel, SnapshotModel, TableModel}
+import bio.terra.workspace.model.DataReferenceDescription
+import bio.terra.workspace.model.DataReferenceDescription.{CloningInstructionsEnum, ReferenceTypeEnum}
+import org.broadinstitute.dsde.rawls.entities.EntityRequestArguments
+import org.broadinstitute.dsde.rawls.mock.{MockDataRepoDAO, MockWorkspaceManagerDAO}
+import org.broadinstitute.dsde.rawls.model.{RawlsUserEmail, RawlsUserSubjectId, UserInfo, Workspace}
+import org.joda.time.DateTime
+import spray.json.{JsObject, JsString}
+
+import scala.collection.JavaConverters._
+
+trait DataRepoEntityProviderSpecSupport {
+
+  // default values for some important attributes
+  val wsid: UUID = UUID.randomUUID()
```

Comment:
Can these two variables (`wsid` and `refid`) be updated to camelCase for consistency?

Reply:
done